examples/utils/webpack-fluid-loader/.eslintrc.cjs

```diff
@@ -7,6 +7,8 @@ module.exports = {
 	extends: [
 		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
 		"prettier",
+		// There are a lot of internal APIs leveraged here.
+		// "../../.eslintrc.cjs",
```

Comment:
We should also probably discuss whether or not we actually want to be publishing this. As it is currently written, I don't think this is an "example". It's also so finely tuned to our test scenarios, that it really shouldn't be a public utility either; it should be a private implementation detail of our "examples" (which I agree with what others have suggested - we should call most of these "test apps" not "examples".

Reply:
At the same time, why shouldn't the external API support what is happening here? (I didn't look into the details. Maybe more of its tricks are above board and not interesting to external parties.)

Reply:
Actually, I think technically they do, and if you connect them both to an external service things will work. The point of webpack-fluid-loader as I understand it was being able to showcase collaboration _while using local server_, which lives in-memory (i.e. the browser tab) so it doesn't support collaboration across browser tabs. And this package was concocted to take an app that makes no special assumptions about how it is being rendered, and renders it twice in the same tab, where both instances can be connected to the local server. I imagine the internal/-ness in this package has more to do with the fact that we want to use local server, than with rendering an app twice in the same tab.

Reply:
Want to make sure we are on the same page about questions/statements.
I am saying that you can't write webpack-fluid-loader without using /internal APIs IFAIK. If the webpack-fluid-loader scenarios are interesting outside of FF repo, then either webpack-fluid-loader should be published to enable these scenarios OR the APIs need exposed to let someone write a thing like webpack-fluid-loader.

If there isn't desire for that, then such things would not be high priority.
Probably there should be a pass over this package to isolate those true /internal APIs and allow just those. That will reveal the special API surface.

Reply:
IMO the long-term goal should be to eliminate webpack-fluid-loader.  It's not representative of how customers use Fluid.  Basically everything outside of the /examples/data-objects directory is the product of slow conversion away from WFL.

The main blockers for completing the transition are:
1. We don't have ODSP support in other examples currently, but should (i.e. `start:spo` script).  Same is true for r11s and docker but I don't think we really care about those as much.
2. There are a variety of utility functionalities it provides that may or may not be useful at this point - we should evaluate whether we care about these (e.g. the extra routes like /pushLogin, /odspLogin, /file*, magic URLs like `#prefetch`, `#binarySnapshot`).  However if useful I suspect these functionalities would be better supported by a standalone dedicated package rather than tying them into the webpack dev server for the example.
3. Labor.  It's not particularly hard to convert from WFL to non-WFL, but it takes some time and testing.